### PR TITLE
Rs 2022 02 20

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 8/08_00_ARC08_2_1.txt [hunk 1]
```diff
@@ -156,32 +177,39 @@ Did this person say 'as a human' just now...?
...
+// I'd assumed ボンボン was more derogatory than it apparently is -
+// 「ぼんぼん」はやや丁寧な言い方だが、育ちが良くて世間知らずだというニュアンスが込められていることもある
+// I guess 'young master' can stay
 "Yes, yes, as you wish... Why did I come all the way to Japan just to babysit this young master..."
 }
 [sha:2892bfd70e04d1dc690dfc295b1cbef63dc88dd2]{
 -- Page 129, Offset 19991.
 -- おかしな二人組は路上駐車させていた乗用車に乗りこむと、駅前方面に走り去っていった。
-The strange duo get in the passenger car that was parked on the road and drive toward the station.
+The strange duo get in a car that was parked by the side of the road,
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_2_1.txt:200]: further evidence this isn't any sort of highway

File: script/Arcueid/Day 8/08_00_ARC08_2_1.txt [hunk 2]
```diff
@@ -191,7 +219,8 @@ It has to be.
 [sha:279ab6d3a6e98282f0fae0d263815462e555a9ca]{
 -- Page 130, Offset 19995.
 -- そもそもあの崩落はアルクェイドのヤツが考えなしにやった事だし、吸血鬼事件と北口駅前の土地は関係がない。
-Besides, that collapse happened because Arcueid wasn't thinking. There's no connection between the vampire incident and the land in front of the north gate.
+Besides, that cave-in was the result of Arcueid not thinking things through.
+ There's no connection between the vampire incidents and the land in front of the north gate.
```
Comment (Avuxo) [inline script/Arcueid/Day 8/08_00_ARC08_2_1.txt:223]: "vampiric incidents" or "killings". I feel like this phrase is used way too much.

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 8/08_00_ARC08_2_1.txt [hunk 3]
```diff
@@ -26,52 +26,67 @@ What business did you have?"
...
-"...Why are you questioning us about that? I know this sounds absurd, but are you implying that the Tohno group were responsible for that fire?"
+"...What is it you want to ask them about?
+ I know this sounds absurd, but are you implying that the Tohno group was responsible for that fire?"
 }
 [sha:c7b06cfc59372f356102eefff39945d642312533]{
 -- Page 120, Offset 19967.
 -- 「うん？　そんな方針は持ってないナイ。ありゃどっから見ても自然発火だし。
-"Hmm? I'm not implying that at all. It was obviously spontaneous combustion no matter how you look at it.
+"Hmm? No, that's not my intent at all, no sir.
+ It was spontaneous combustion no matter which way you slice it.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_2_1.txt:62]: It was spontaneous combustion, **pure and simple**?

File: script/Arcueid/Day 8/08_00_ARC08_2_1.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_2_1.txt:223]: yeah it really is, and it's really awkward literally

File: script/Arcueid/Day 8/08_00_ARC08_2_1.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 8/08_00_ARC08_2_1.txt:62]: Sounds good to me

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 4]
```diff
@@ -126,37 +140,42 @@ In that case, how lucky. Super lucky. Not for me, but for you. Don't you want so
...
-I feel like it's the kind of serious where he'll take great care not to miss a single piece of info in anything I say, no matter how trifling.
+I get the impression that he's paying close attention to my every word,
+ no matter how trifling,
+ careful not to miss any cues as to my attitude.
 }
 [sha:cd132564562e6682059b4398df643ecacba2070b]{
 -- Page 104, Offset 19927.
 -- 「あ、もしかして野次馬だと思われてる？
-"Ah, did you think that I was some sort of curious onlooker?
+"Ah, you think I'm some sort of nosy parker don't you?
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_2B.txt:165]: Curious onlooker doesn't have as much of the mildly intrusive/probing vibe, thoughts on `[nosy] busybody`?

File: script/Arcueid/Day 8/08_00_ARC08_2_1.txt [hunk 5]
```diff
@@ -156,32 +177,39 @@ Did this person say 'as a human' just now...?
...
 -- Page 128, Offset 19989.
 -- オラ、帰るぞアンドー。いいかげん食事時だ、ケーキ食わせろ、ケーキ。チーズと蜂蜜たっぷりなヤツな！」
-Hey, we're leaving Ando. It's time to eat. I want cake, you hear me? One that's full of cheese and honey!"
+Hey, we're leaving Ando. I've had enough, it's time to eat.
+ I want cake, you hear me? One that's full of cheese and honey!"
 }
 [sha:5e5ca9d7c6b5db343bb51ca47b8c22805b93be9c]{
 -- Page 128, Offset 19990.
 -- 「へえへえ、仰せのままに。……なーんでオレ、日本に来てまでボンボンのお守りしてんだろうなぁ……」
+// I'd assumed ボンボン was more derogatory than it apparently is -
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_2_1.txt:192]: Yeah, spoiled brat is what I'd originally replaced it with. Suggestion works for me.

File: script/Arcueid/Day 8/08_00_ARC08_2_1.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_2_1.txt:62]: I should start keeping a list of these

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt [hunk 6]
```diff
@@ -126,37 +140,42 @@ In that case, how lucky. Super lucky. Not for me, but for you. Don't you want so
...
-Rather it was so serious that it was frightening.
+It's more that I'm terrified by how intent he is.
 }
 [sha:35ce727002b5ac108b4264549f23a3fc36cbf1a1]{
 -- Page 103, Offset 19926.
 -- なんでもない言葉の中に、こちらの態度を一つも取りこぼさないよう注目する<本|・><気|・>を感じたというか……。
-I feel like it's the kind of serious where he'll take great care not to miss a single piece of info in anything I say, no matter how trifling.
+I get the impression that he's paying close attention to my every word,
+ no matter how trifling,
+ careful not to miss any cues as to my attitude.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 8/08_00_ARC08_2B.txt:160]: I think I had intentionally gone with cue, but if we think clue is better can change

File: script/Arcueid/Day 8/08_00_ARC08_2B.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 8/08_00_ARC08_2B.txt:165]: busybody is alright

**Pull request merged**